src/lib/eina/eina_debug.c

```diff
@@ -704,14 +707,16 @@ eina_debug_session_data_get(Eina_Debug_Session *session)
 Eina_Bool
 eina_debug_init(void)
 {
-   pthread_t self;
+   
+   Eina_Thread self;//pthread_t self;
```

Comment:
There is no reason to comment the old version, just delete it. And another thing, why the new line?